packages/performance-tests/test/TransformerRegression.test.ts

```diff
@@ -102,17 +107,60 @@ async function runRegressionTests() {
   describe("Transformer Regression Tests", function () {
     testIModels.forEach(async (iModel) => {
       describe(`Transforms of ${iModel.name}`, async () => {
+        let sourceDb: BriefcaseDb;
+        let reportInfo: ReporterInfo
+
+        before(async () => {
+          Logger.logInfo(loggerCategory, `processing iModel '${iModel.name}' of size '${iModel.tShirtSize.toUpperCase()}'`);
+          sourceDb = await iModel.load();
+          const fedGuidSaturation = sourceDb.withStatement(
+            `
+            SELECT
+            CAST(SUM(hasGuid) as DOUBLE)/SUM(total) ratio 
```

Comment:
instead of `SUM(total)` can't you just do `COUNT(*)` and remove the `total` column from the query?

Reply:
done